Main/ClientApp/src/Components/Drawer/MyCollectionsSection.tsx

```diff
@@ -130,9 +130,6 @@ export default function MyCollectionsSection() {
                       </ListItemIcon>
                       <ListItemText primary="Get sharable link" />
                     </ListItem>
-                    {/* //<></>}
-                    /*
-                  // @ts-ignore */}
                     <ListItem
```

Comment:
Does it work without the ts-ignore comment?

Reply:
it works on my machine ;)